packages/wrangler/src/__tests__/mock-cfetch.ts

```diff
@@ -11,7 +11,7 @@ export type MockHandler<ResponseType> = (
   uri: RegExpExecArray,
   init: RequestInit,
   queryParams: URLSearchParams
-) => ResponseType;
+) => ResponseType | Promise<ResponseType>;
```

Comment:
Is it hard to make this uniformly async?
```suggestion
) => Promise<ResponseType>;
```

Reply:
Most of the time we do not need these handlers to be async. This way, the test write can just choose which is best for them.